operator/internal/deploy/webhook_config_test.go

```diff
@@ -58,14 +58,14 @@ var _ = Describe("deploy watcher", Ordered, func() {
 	})
 
 	It("deploys watcher helm chart with correct webhook config", func() {
-		err := deploy.UpdateWebhookConfig(ctx, webhookChartPath, watcherCR, testEnv.Config, k8sClient)
+		err := deploy.UpdateWebhookConfig(ctx, webhookChartPath, watcherCR, testEnv.Config, k8sClient, "")
```

Comment:
we should have an explainable variable to the last param, like `"fake-ip"` instead of empty string.